src/main/java/core/basesyntax/SetFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface SetFigure {
```

Comment:
1. What do you need this interface for?
2. https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-verbs-for-classinterface-names